pages/index.tsx

```diff
@@ -45,27 +46,24 @@ export default function Home() {
                 Playwright support
               </a>
             </Chip>
-            <h1 className="text-4xl sm:leading-tight sm:text-6xl font-bold bg-clip-text">
+            <h1 className="text-4xl sm:leading-tight sm:text-6xl font-bold bg-clip-text max-w-2xl mx-auto">
               <span className="bg-clip-text text-transparent bg-gradient-to-b from-white to-[#AAAAAA]">
-                Detect bugs
```

Comment:
If you change it you have to change it everywhere:

- Marketplace
- Product Hunt
- Twitter
- etc...

- Social image

It is like a "rebranding".

Reply:
We seem to have different slogans across our platforms. I'm opening a ticket to unify them and keep our messaging consistent.

Reply:
https://github.com/orgs/argos-ci/projects/1/views/1?pane=issue&itemId=32807419